### DISCUSSION TITLE
你想这个框架扩展那些功能？

### DISCUSSION THREAD

**Discussion by Xutaotaotao** [Xutaotaotao/electron-prokit#36] [Ideas]
可以写下你的需求，帮你实现！！

**Comment by qian-kong** [comment]
你好，我大概看了下启动方式，这里我有个疑问：
main部分是先build到dist文件之后再执行main/index.js的代码来启动electron，如果我要调试主进程里面的代码，只能通过console.log的方式，而vscode下调试模式添加断点的方式貌似不行

**Reply by Xutaotaotao** [reply]
这个也可以做一下vs code 的配置，后期也会加，现在核心的东西还没做好，加一个todo。

**Reply by qian-kong** [reply]
ok
